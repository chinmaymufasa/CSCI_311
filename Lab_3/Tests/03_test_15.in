2
106
5
2
71
5
2
21
5
2
0
5
2
139
5
2
15
5
2
70
5
2
126
5
2
146
5
2
7
5
2
172
5
2
143
5
2
111
5
2
114
5
2
69
5
2
58
5
2
156
5
2
49
5
2
154
5
2
2
5
-1
