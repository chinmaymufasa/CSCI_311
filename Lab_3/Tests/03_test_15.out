106 
71 106 
21 71 106 
0 21 71 106 
0 21 71 106 139 
0 15 21 71 106 139 
0 15 21 70 71 106 139 
0 15 21 70 71 106 126 139 
0 15 21 70 71 106 126 139 146 
0 7 15 21 70 71 106 126 139 146 
0 7 15 21 70 71 106 126 139 146 172 
0 7 15 21 70 71 106 126 139 143 146 172 
0 7 15 21 70 71 106 111 126 139 143 146 172 
0 7 15 21 70 71 106 111 114 126 139 143 146 172 
0 7 15 21 69 70 71 106 111 114 126 139 143 146 172 
0 7 15 21 58 69 70 71 106 111 114 126 139 143 146 172 
0 7 15 21 58 69 70 71 106 111 114 126 139 143 146 156 172 
0 7 15 21 49 58 69 70 71 106 111 114 126 139 143 146 156 172 
0 7 15 21 49 58 69 70 71 106 111 114 126 139 143 146 154 156 172 
0 2 7 15 21 49 58 69 70 71 106 111 114 126 139 143 146 154 156 172 
